unison-src/transcripts/names.output.md

```diff
@@ -1,36 +1,105 @@
- Example uses of the `names` command and output
+# `names` command
+
+Example uses of the `names` command and output
+
 ```unison
-structural type IntTriple = IntTriple (Int, Int, Int)
-intTriple = IntTriple(+1, +1, +1)
+-- Some names with the same value
+some.place.x = 1
+some.otherplace.y = 1
+some.otherplace.x = 10
+somewhere.z = 1
+-- Some similar name with a different value
+somewhere.y = 2
 ```
 
 ```ucm
-.> alias.type IntTriple namespc.another.TripleInt
 
-  Done.
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      some.otherplace.x : Nat
+      some.otherplace.y : Nat
+      some.place.x      : Nat
+      somewhere.y       : Nat
+      somewhere.z       : Nat
 
-.> alias.term intTriple namespc.another.tripleInt
+```
+```ucm
+.> add
 
-  Done.
+  ⍟ I've added these definitions:
+  
+    some.otherplace.x : Nat
+    some.otherplace.y : Nat
+    some.place.x      : Nat
+    somewhere.y       : Nat
+    somewhere.z       : Nat
 
-.> names IntTriple
+```
+`names` searches relative to the current path.
 
-  Type
-  Hash:  #cp7a2qo5du
-  Names: IntTriple namespc.another.TripleInt
+```ucm
+-- We can search by suffix and find all definitions named 'x', and each of their aliases respectively.
+-- But we don't see somewhere.z which is has the same value but is out of our namespace
```

Comment:
Wouldn't it be better to have this text outside the `ucm` block?  Possibly a general question about comments in `ucm` blocks.

Reply:
I suppose that's personal preference, I personally think doing it this way negatively affects readability, and in cases like here where we're running several commands in a row it lets us document each command alongside the command itself, which saves a bit of visual noise and boilerplate rather than split the codeblock up into a bunch of separate codeblocks.